css/elements/image.css

```diff
@@ -0,0 +1,22 @@
+@import "../utilities/variables.css";
+
+.img{
+    max-width: 100%;
+    height: auto;
+    object-fit: fill;
```

Comment:
fill can squeeze the image , you can also use the property cover.

Reply:
Hey @lannisterrr I have change object-fit property to cover.